### PR TITLE
[DSS-306] add dark mode to Storybook

### PR DESCRIPTION
**Pull request by QuintonJason**
# Description
This is a followup PR to https://github.com/Kajabi/sage/pull/18 as there was an error blocking the initial work
- [x] Add dark mode functionality to storybook
- [x] Add dark theme to storybook

|  Light  |  Dark  |
|--------|--------|
|Screen Shot 2023-02-15 at 1 26 11 PM|Screen Shot 2023-02-15 at 1 26 19 PM|

**Dependencies**
- `storybook-dark-mode`
- `babelOptions`

Fixes #(issue)
[DSS-306](https://kajabi.atlassian.net/browse/DSS-306)

## Type of change
- [x] This change requires a documentation update

# How Has This Been Tested?
No tests provided, docs only

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

[DSS-306]: https://kajabi.atlassian.net/browse/DSS-306?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/core/package.json [hunk 1]
```diff
@@ -50,11 +50,13 @@
     "test.watch": "stencil test --spec --e2e --watchAll"
   },
   "dependencies": {
-    "@stencil/core": "^2.18.0",
-    "@sage/icons": "*"
+    "@babel/plugin-syntax-jsx": "^7.18.6",
```
Comment (ju-Skinner) [inline libs/core/package.json]: this needs to be under `devDependencies`. You can just cut and paste it and re-run `yarn setup`. Be sure to commit the `yarn.lock` as well.

Comment (ju-Skinner) [reply libs/core/package.json]: Looks like you have it in here twice. Just remove the one under `dependencies`.

**Review by ju-Skinner — CHANGES_REQUESTED** [review]
See comments about the dependencies.

File: libs/core/package.json (hunk 1 above)
Comment (QuintonJason) [reply libs/core/package.json]: Updated. No `yarn.lock` change since this already existed in `devDependencies`

**Comment by goodwinchris** [comment]
This appears to be working well but some of our components don't work on a dark canvas, should we be styling canvases to be white even in dark mode? 
CleanShot 2023-02-22 at 10 47 44@2x

**Comment by QuintonJason** [comment]
Putting this PR into Draft as there are additional complications with the style switching. We'll revisit this after discovery with Storybook Doc 7 beta

**Comment by QuintonJason** [comment]
Closing this PR as Storybook now is updated to Storybook 7

**Pull request closed without merging**